rook/Tester.py

```diff
@@ -221,12 +226,28 @@ def _get_gold_files(self):
     """
       returns a list of the full path to the gold files
       @ In, None
-      @ Out, _get_gold_files, List(Strings), the path of the gold files.
+      @ Out, paths, List(Strings), the paths of the gold files.
     """
-    if len(self.specs['gold_files']) > 0:
+    this_os = platform.system().lower()
+    available_os = ['windows', 'mac', 'linux'] # list of OS with specific gold file options
+
+    # replace "darwin" with "mac"
+    if this_os == 'darwin':
+      this_os = 'mac'
+
+    # check if OS specific gold files should be used
+    if (this_os in available_os) and (len(self.specs[f'{this_os}_gold']) > 0):
+      gold_files = self.specs[f'{this_os}_gold'].split()
+      paths = [os.path.join(self.__test_dir, f) for f in gold_files]
+    # if OS specific gold files are not given, are specific gold files given?
+    elif len(self.specs['gold_files']) > 0:
       gold_files = self.specs['gold_files'].split()
-      return [os.path.join(self.__test_dir, f) for f in gold_files]
-    return [os.path.join(self.__test_dir, "gold", f) for f in self.__output_files]
+      paths = [os.path.join(self.__test_dir, f) for f in gold_files]
+    # otherwise, use output files
```

Comment:
I would like to suggest to use relative path to gold directory. @joshua-cogliati-inl Any suggestion here?

Reply:
There are a few tests that would need to have gold files moved to the gold directory to make this happen. I think it is probably better to do that and make a minor change here so that all gold files are in the gold directory.

Reply:
I am fine with it being relative to the test directory.

Reply:
If possible, I would like to have a consistent assumption about the path for the gold files.

Reply:
> If possible, I would like to have a consistent assumption about the path for the gold files.

Okay, I will change them to be relative to the "gold" directory. I may need to move some gold files around or modify tests to place files in the "gold" directory, but it will all be consistent.

Reply:
Hm, changing them to be relative to the "gold" directory breaks the HERON plugin tests, so they will need to be fixed.

Reply:
(Sorry, but changes seem to propagate :( )

Reply:
Correct me if I'm wrong, but it seems to me that we only assume relative to `gold` if you do NOT specify a custom path to the gold file; however, if you assign a custom path, it assumes relative to the test directory. Is that right?

Reply:
@PaulTalbot-INL, yes, that is how it was set up originally. My most recent commit changes this to assume all given paths are relative to `gold`. This change did not affect most tests. The ARMA reseed precursor test required some "creative" path manipulations to get it to work where all gold files are relative to `gold`, but now all RAVEN tests pass. It looks like there is one HERON test that needs to be modified, but other plugins seem to be unaffected by this change.